run_sims/sweeps/historical_archetype_sweeps.py

```diff
@@ -0,0 +1,62 @@
+from functools import partial
+
+from run_sims.archetypes.magude.magude_campaign import build_full_magude_campaign
+from run_sims.archetypes.maka.maka_campaign import build_full_maka_campaign
```

Comment:
maka_campaign is not checked in this PR.

Reply:
I see maka_campaign.py in my branch.. can you double check that you don't have it in yours?

Reply:
I can see it now, thank you Josh!